### ISSUE TITLE
X and Y axis inversion when drawing a circle using the circle function

### ISSUE THREAD

**Issue by pablo-souza** [rainyl/opencv_dart#284]
**Describe the bug**
X and Y axis inversion when drawing a circle using the circle function

**To reproduce**
```dart
alternatives = //image
gray = await cv.cvtColorAsync(alternatives, cv.COLOR_BGR2GRAY);
blur = await cv.gaussianBlurAsync(gray, (3, 3), 0,);
thresh = (await cv.thresholdAsync(blur, 150, 255, cv.THRESH_BINARY)).$2;
final contornos = (await cv.findContoursAsync(
    thresh, cv.RETR_LIST, cv.CHAIN_APPROX_NONE))
    .$1;

for (var contour in contornos) {
  var area = cv.contourArea(contour);
  var perimeter = cv.arcLength(contour, true);
  var circularidade = 4 * 3.14159 * (area / (perimeter * perimeter));
  if (0.7 < circularidade && circularidade <= 1.2){
    var (center,radius) = cv.minEnclosingCircle(contour);
    int ratio = radius.toInt();
    int x = center.x.toInt();
    int y = center.y.toInt();
    cv.circle(
      alternatives,
      cv.Point(x,y),
      ratio, cv.Scalar.red,
    );
    cont++;
  }
}

return cv.imencode(".png", alternatives).$2;
```

**Expected behavior**
Draw a circle around each alternative
![photo_2024-11-02_12-14-44](https://github.com/user-attachments/assets/691608f6-efdc-4c97-af50-c32f3945bea8)

**Found behavior**
A single circle was drawn
![photo_2024-11-02_12-14-40](https://github.com/user-attachments/assets/c2701a18-1d1d-4340-a164-c7c22abbed40)

**Solution**
```dart
alternatives = //image
gray = await cv.cvtColorAsync(alternatives, cv.COLOR_BGR2GRAY);
blur = await cv.gaussianBlurAsync(gray, (3, 3), 0,);
thresh = (await cv.thresholdAsync(blur, 150, 255, cv.THRESH_BINARY)).$2;
final contornos = (await cv.findContoursAsync(
    thresh, cv.RETR_LIST, cv.CHAIN_APPROX_NONE))
    .$1;

for (var contour in contornos) {
  var area = cv.contourArea(contour);
  var perimeter = cv.arcLength(contour, true);
  var circularidade = 4 * 3.14159 * (area / (perimeter * perimeter));
  if (0.7 < circularidade && circularidade <= 1.2){
    var (center,radius) = cv.minEnclosingCircle(contour);
    int ratio = radius.toInt();
    int x = center.x.toInt();
    int y = center.y.toInt();
    cv.circle(
      alternatives,
      cv.Point(y,x), //inversion
      ratio, cv.Scalar.red,
    );
    cont++;
  }
}

return cv.imencode(".png", alternatives).$2;
```

**Comment by rainyl** [comment]
verified, will be fixed soon.

**Comment by rainyl** [comment]
@pablo-souza This bug has been resolved in v1.3.2, you can upgrade now~
